src/nnet3/decodable-batch-looped.h

```diff
@@ -0,0 +1,357 @@
+// nnet3/decodable-batch-looped.h
+
+// Copyright 2020 XiaoMi.crop (author: yanzhao)
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef KALDI_NNET3_DECODABLE_BATCH_LOOPED_H_
+#define KALDI_NNET3_DECODABLE_BATCH_LOOPED_H_
+
+#if HAVE_CUDA == 1
+#include <queue>
```

Comment:
Headers from the standard library are usually sorted alphabetically.

Reply:
ok